x/oracle/keeper/operator_test.go

```diff
@@ -0,0 +1,100 @@
+package keeper_test
+
+import (
+	"testing"
+	"time"
+
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	"github.com/certikfoundation/shentu/simapp"
+
+	"github.com/stretchr/testify/require"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/certikfoundation/shentu/x/oracle/types"
```

Comment:
let's sort these imports. the order is based on dependency.

Reply:
I've initially had sorted them similarly to tests from other modules, but I kept getting linting errors.

Reply:
you'll have to group each of them together with a blank line.